src/generators/app-lit-element-ts/templates/_package.json

```diff
@@ -2,18 +2,16 @@
   "name": "<%= tagName %>",
   "license": "MIT",
   "scripts": {
-    "start": "concurrently --kill-others --names tsc,web-dev-server \"npm run tsc:watch\" \"web-dev-server --app-index index.html --node-resolve --open --watch\"",
-    "tsc:watch": "tsc --watch"
+    "start": "tsc && concurrently -k -r \"tsc --watch --preserveWatchOutput\" \"wds\""
```

Comment:
Is `-k` the se as `--kill-others`?

What was `--names` giving us?

Reply:
`-k` is the same as `--kill-others` yes. Trying to keep it shorter.

`--names` gives prefixes to the terminal output, but when using `--raw` the output is no longer prefixed